pycltheme/static/css/cl.css

```
@@ -237,26 +240,156 @@ blockquote {
 
 
 /* Botón Discord Python en Español */
-.discord {
-}
 
 .discord button {
   width: 100%;
   justify-content: center;
 }
 
-.discord .btn-primary,
-.discord .btn-primary:active,
-.discord .btn-primary:visited,
-.discord .btn-primary:focus {
+ .btn-default,
+ .btn-default:active,
+ .btn-default:visited,
+ .btn-default:focus {
   background-color: #132e45;
   border-color: white;
+  color:white;
+  display: flex;
+  align-items: center;
 }
-.discord .btn-primary:hover {
+ .btn-default:hover {
   background-color: #859bee;
   border-color: #859bee;
+  text-decoration: none;
 }
 
 .discord a {
   text-decoration: none;
 }
+
+altura{
```

```suggestion
altura {
```
¿Podemos quizás cambiar este nombre para seguir la tónica de bootstrap y quzás usar `h-100-responsive`?

---

ok, aunque tal vez elimine esa clase, pero entiendo el punto